stylus/cozy-ui/defaults.styl

```diff
@@ -32,7 +32,20 @@ textarea
 
 // UTILS
 @import '../ui-base/mixins'
+@import '../ui-base/palette'
 :global
+    .coz-error
+        color  pomegranate
+
+        &.coz-error--warning:before
```

Comment:
aren't we exposing too much common classes in global namespace (true question) ?